Library/Formula/ipbt.rb

```diff
@@ -2,9 +2,8 @@
 
 class Ipbt < Formula
   homepage 'http://www.chiark.greenend.org.uk/~sgtatham/ipbt/'
-  url 'http://www.chiark.greenend.org.uk/~sgtatham/ipbt/ipbt-r9487.tar.gz'
-  sha1 'cd7c0b991a9422ed5f31ce13846f28fb5860b0dc'
-  version 'r9487'
+  url 'http://www.chiark.greenend.org.uk/~sgtatham/ipbt/ipbt-20141026.2197432.tar.gz'
```

Comment:
Not sure why the different versioning scheme, but this appears to be the correct version. The old one is 404.